code/modules/antagonists/blob/structures/_blob.dm

```diff
@@ -104,6 +104,9 @@
 
 /obj/structure/blob/proc/ConsumeTile()
 	for(var/atom/A in loc)
+		if(iscarbon(A) && overmind) // Make sure to inject strain-reagents with automatic attacks when needed.
```

Comment:
Why are you checking for carbons when you're using `attack_living`? This, and the other place, should probably be `iscarbon(A) && !isblobmonster(A)`

Reply:
Iscarbon because there are certain living mobs (revenant etc) that just call return on blob_act because they want nothing to do with it, and injecting them with chems might have unintended consequences.